test/webscript.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+var path = require('path');
+var assert = require('yeoman-assert');
+var helpers = require('yeoman-generator').test;
+
+describe('generator-alfresco:webscript', function () {
```

Comment:
Yes in as much as it needs to be filled out! So keeping ING the file is my
shame and reminder that it needs to happen.